pb/node.proto

```diff
@@ -10,6 +10,11 @@ enum SyncState {
   PERSIST_FINISHED = 3;
 }
 
+enum LedgerMode {
+  full = 0;
```

Comment:
Full synchronization contains two modes: full sync and fast sync, do we need to distinguish them?

Reply:
I don't think so. They are identical once sync finished